Source/WebCore/rendering/style/RenderStyleInlines.h

```diff
@@ -670,6 +670,7 @@ inline bool RenderStyle::specifiesColumns() const { return !hasAutoColumnCount()
 constexpr OptionSet<Containment> RenderStyle::strictContainment() { return { Containment::Size, Containment::Layout, Containment::Paint, Containment::Style }; }
 inline StyleColor RenderStyle::strokeColor() const { return m_rareInheritedData->strokeColor; }
 inline float RenderStyle::strokeMiterLimit() const { return m_rareInheritedData->miterLimit; }
+inline const AtomString& RenderStyle::styleTypeIdentifier() const { return m_nonInheritedData->rareData->styleTypeIdentifier; }
```

Comment:
This should be called something more informative, either something specific to this feature like `viewTransitionPseudoElementNameSelector` or something more generic like `functionalPseudoElementArgument`. Of course `styleType()` is weird in itself, it should be `pseudoElementType()` or similar.

Reply:
will change to `functionalPseudoElementArgument` . I'll rename `styleType()` in a separate patch.